quickstart-node/src/utils/url.utils.ts

```diff
@@ -0,0 +1,9 @@
+export const generateUrlSafeString = (length: number) => {
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_'; // URL-safe characters
```

Comment:
Is there a solution without creating a var with all the characters?

Reply:
Yes, it includes installing a package called `crypto` and it is a heavy library. So I just created the function my self